apps/shooting-clubs/controllers/confirm.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const Controller = require('../../common/controllers/confirm');
+
+module.exports = class ConfirmController extends Controller {
+
+  getFieldData(field, req) {
+    const defaults = super.getFieldData(field, req);
+
+    if (field.field === 'location-addresses') {
+      return defaults.value.map(address => {
+        address['address-category'] = [].concat(address['address-category']);
```

Comment:
If you only select a single checkbox then the value is a string. This makes sure it's always an array.